project/BuildHelper.scala

```diff
@@ -8,26 +8,12 @@ import sbtcrossproject.CrossPlugin.autoImport._
 import scalafix.sbt.ScalafixPlugin.autoImport._
 
 object BuildHelper {
-  private val versions: Map[String, String] = {
-    import org.snakeyaml.engine.v2.api.{Load, LoadSettings}
+  val Scala211: String = "2.11.12"
+  val Scala212: String = "2.12.16"
+  val Scala213: String = "2.13.8"
+  val Scala3: String   = "3.2.0"
```

Comment:
Back to Build helper as source of truth?

Reply:
Yeah... In this specific situation, there was no clear advantage to keeping it in GitHub Actions, and so I've moved it back to sbt.